info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 2    | 2.1.3    | KDOC_NO_NEWLINES_BETWEEN_BASIC_TAGS       | Check: if there is a newline of an empty KDoc line (with a leading asterisk) between `@param`, `@return`, `@throws` tags<br>Fix: removes line                                                                                                                       | yes | no                                                                                                                                                                                                                         |                                                                                                                                                                                                                               |
+| 2    | 2.1.3    | KDOC_NO_NEWLINE_AFTER_SPECIAL_TAGS        | Check: warns if special tags `@apiNote`, `@implNote`, `@implSpec` don't have exactly one empty line after<br>Fix: removes redundant lines or adds one                                                                                                        | yes | no                                                                                                                                                                                                                         | Handle empty lines without a leading asterisk                                                                                                                                                                                   |
+| 2    | 2.1.3    | KDOC_NO_DEPRECATED_TAG                    | Check: warns if `@deprecated` is used in KDoc<br>Fix: adds `@Deprecated` annotation with a message, removes tag                                                                                                                                                | yes | no                                                                                                                                                                                                                         | The `replaceWith` field in the annotation can be filled too                                                                                                                                                                            |
```

Comment:
"The `replaceWith` field in the annotation can be filled too." Is this statement correct?

Reply:
We should consider not only annotating a class/method/property with `@Deprecated`, but also provide a meaningful `replaceWith` value, e.g.:

```kotlin
@Deprecated(replaceWith = "the code to be used as a replacement")
fun f() {
}
```

Reply:
Changed:

The `replaceWith` field in the annotation can also be filled with a meaningful value.